requirements/prod.txt

```diff
@@ -45,23 +45,23 @@ bleach[css]==6.1.0 \
     --hash=sha256:0a31f1837963c41d46bbf1331b8778e1308ea0791db03cc4e7357b97cf42a8fe \
     --hash=sha256:3225f354cfc436b9789c66c4ee030194bee0568fbf9cbdad3bc8b5c26c5f12b6
     # via -r requirements/prod.in
-boto3==1.34.131 \
-    --hash=sha256:05e388cb937e82be70bfd7eb0c84cf8011ff35cf582a593873ac21675268683b \
-    --hash=sha256:dab8f72a6c4e62b4fd70da09e08a6b2a65ea2115b27dd63737142005776ef216
+boto3==1.34.144 \
```

Comment:
I typically only look at the top level and I'm looking for 2 indicators mostly:
1. Is there an update to an "important" package we use.
2. How far behind are we? Are package updates building up and we should update them soon?

I think as long as we can infer that from dependabot it's helpful. Whether it updates prod.txt vs prod.in doesn't matter since we run them manually.

Reply:
We've basically settled on the same with @stevejalim — using that as a mere reminder, doing the `make compile-requirements` on our own anyways…

Reply:
It's actually the culprit I think…  Be it pip-compile-multi or custom make wizardry — I think https://bedrock.readthedocs.io/en/latest/architecture/decisions/0006-revise-tooling-for-python-dependency-management.html#consequences is still valid, i.e. dependabot can't "exec" stuff so they don't know what's recursive where how etc. to effectively output the requirements "computed" completely from the inputs. They don't even seem to see the relation between input and output… but still. If they can update the locked dep in foo.txt I'd still expect them blindly updating it in bar.txt as well in the same fashion, even if the bot can't understand the relation of the files or what depends on what…

(I at least wanted for the grouped PR to run the CI in one lump to see the checks for rough indication if there's something that needs extra attention etc.…)